src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class FileWork {
+    public static final String CHARACTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
```suggestion
        StringBuilder stringBuilder = new StringBuilder();
```
redundant empty line